packages/ui-popover/src/Popover/README.md

```diff
@@ -105,21 +152,136 @@ render: false
 example: true
 ---
 class Example extends React.Component {
-render () {
-  return (
-      <div style={{ paddingBottom: 25, display: 'flex', justifyContent: 'center' }}>
-        <Popover
-          renderTrigger={<div style={{display: 'inline-block', height: '3px', width: '3px', background: 'blue'}}/>}
-          isShowingContent={true}
-          placement="end top"
-          shouldAlignArrow
-          mountNode={() => document.getElementById('main')}
-        >
-          <Heading>Small<br/>Target</Heading>
-        </Popover>
-      </div>
-  )
-}
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            shouldAlignArrow: true,
+            isShowingContent: true,
+            withArrow: true,
+            placement: 'top',
+            shadow: 'topmost',
+            placementsValues: [
+                'top',
+                'end',
+                'bottom',
+                'start',
+                'top start',
+                'start top',
+                'start center',
+                'start bottom',
+                'bottom start',
+                'bottom center',
+                'bottom end',
+                'end bottom',
+                'end center',
+                'end top',
+                'top end',
+                'top center',
+                'center end',
+                'center start'
+            ],
+            shadowValues: [
+                'none',
+                'resting',
+                'above',
+                'topmost'
+            ],
+        }
+    }
+    changePlacement = (e, { id, placement }) => this.setState({ placement: this.state.placementsValues[id] })
```

Comment:
Simple select passes to its onchange function 2 parameters. The first is indeed an event, the second is always {value:"",id:""} shape. You access placement and shadow, (which are not in the second parameter, and you don't use them in the function body. Since you have access to the value, you can do something like this:
```suggestion
    changePlacement = (e, { value }) => this.setState({ placement: value})
```